eo-maven-plugin/src/test/java/org/eolang/maven/AssembleMojoTest.java

```diff
@@ -86,7 +86,7 @@ void assemblesTogether(@TempDir final Path temp) throws Exception {
                     )
                 )
             ),
-            Matchers.is(new Online().value())
+            Matchers.is(true)
```

Comment:
@volodya-lombrozo Why do we need this change?

Reply:
@Graur We already have `@ExtendWith(OnlineCondition.class)` statement which is declared for `AssembleMojoTest`. It means, if `new Online().value() == false`, then the tests won't be invoked. Thus, we can just put the `true`. Moreover, by removing `new Online().value()` we can hide this class inside `OnlineCondition` class.

Reply:
@volodya-lombrozo Thanks!